includes/class-kadence-blocks-prebuilt-library-rest-api.php

```diff
@@ -588,6 +607,71 @@ public function get_remaining_credits( $request ) {
 			return rest_ensure_response( $response );
 		}
 	}
+	/**
+	 * Handle the event and maybe send to Prophecy WP.
+	 */
+	public function handle_event( $request ) {
+		$event_label = $request->get_param( self::PROP_EVENT_LABEL );
+		$event_data  = $request->get_param( self::PROP_EVENT_DATA );
+		$event_data  = json_decode( $event_data, true );
```

Comment:
If you make the JSON schema for this an array/array of objects, you won't have to decode this at all, [example](https://github.com/stellarwp/prophecy-wordpress/blob/735a5281e92ee8ca43c4860c8dd2c453fa5e7200/src/Rest/Content/Controllers/Content.php#L162-L181).